### DISCUSSION TITLE
Possibilité d'attribuer des valeurs maximales pour les caractéristiques

### DISCUSSION THREAD

**Discussion by Seikame** [fibreville/minotaure_php#72] [Boite à idées]
Je pense que pour certains types de parties ça pourrait être intéressant de plafonner à un maximum donné les variables comme les points de vie ou les autres caractéristiques, histoires que des joueurs chanceux ne s'envolent pas au niveau des valeurs en cas de successions d'évènements positifs. 

On pourrait évidemment imaginer l'opposé avec des valeurs minimales sous lesquelles les personnages ne pourraient pas descendre.

J'imagine que ça pourrait être d'autant + intéressant voir indispensable si plusieurs systèmes sont déclinés pour la suite, avec D20, D100, ou si on commence à vouloir proposer du roll over.

**Comment by ElethornGM** [comment]
Très bonne idée, surtout en terme de gains de PV par exemple, mais aussi pour les caractéristiques. 
Il pourrait y avoir dans les options, au même endroit où l'on peut renommer les caractéristiques, la possibilité de renseigner une valeur max et une valeur min.
